### PR TITLE
Feature/pong

### PR DESCRIPTION
**Pull request by Ulysseclem**
Pong view + good test

### CODE REVIEW

File: frontend/src/static/js/views/Pong.js [hunk 1]
```diff
@@ -6,10 +6,179 @@ export default class Pong extends Component {
...
+
+	getScript()
+	{
+		const js = `
+		var leftPaddle;
+		var rightPaddle;
+		var	BallVar;
+		var	isPlaying = false;
+
+		function startGame() {
```
Comment (cdurdetrouver) [inline frontend/src/static/js/views/Pong.js:41]: you can refactor this

File: frontend/src/static/js/views/Pong.js [hunk 2]
```diff
@@ -6,10 +6,179 @@ export default class Pong extends Component {
...
+			myGameArea.start();
+			leftPaddle = new Paddle(30, 105, "w", "s");
+			rightPaddle = new Paddle(440, 105, "ArrowUp", "ArrowDown");
+			BallVar = new Ball();
+			window.addEventListener('keydown', function(e) {
```
Comment (cdurdetrouver) [inline frontend/src/static/js/views/Pong.js:46]: same

**Review by cdurdetrouver — APPROVED** [review]

File: frontend/src/static/js/views/Pong.js (hunk 1 above)
Comment (Ulysseclem) [reply frontend/src/static/js/views/Pong.js:41]: tg

File: frontend/src/static/js/views/Pong.js (hunk 2 above)
Comment (Ulysseclem) [reply frontend/src/static/js/views/Pong.js:46]: ntm

**Pull request merged**